### PR TITLE
Fix crashes reported to errors.ubuntu.com and workaround for debootstrap failing

### PR DESCRIPTION
**Pull request by rbalint**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -12,7 +12,7 @@ Build-Depends: debhelper (>= 9.20160709),
                python3-setuptools
 Build-Depends-Indep: python3-dev,
                      python3-coverage,
-                     pep8,
+                     pycodestyle,
```
Comment (mvo5) [inline debian/control]: pycodestyle looks like a nice enhancement. I see two potential problems: 
1. Its only available in bionic and later so backporting u-u becomes harder.
2. Its in universe but I guess that is a problem you will deal with :)

File: unattended-upgrade [hunk 2]
```diff
@@ -1403,23 +1403,28 @@ def get_dpkg_log_content(logfile_dpkg, install_start_time):
         logfile_dpkg, install_start_time))
     content = []
     found_start = False
-    with io.open(logfile_dpkg, encoding='utf-8', errors='replace') as fp:
```
Comment (mvo5) [inline unattended-upgrade]: Nice catch

File: unattended-upgrade-shutdown [hunk 3]
```diff
@@ -89,7 +89,11 @@ def signal_stop_unattended_upgrade():
     if os.path.exists(pidfile):
         pid = int(open(pidfile).read())
         logging.debug("found running unattended-upgrades pid %s" % pid)
-        os.kill(pid, signal.SIGTERM)
+        try:
```
Comment (mvo5) [inline unattended-upgrade-shutdown]: Nice one!

File: debian/control (hunk 1 above)
Comment (rbalint) [reply debian/control]: @mvo5 since Travis was also failing I added a fallback, this will make backporting easier.

**Pull request merged**